etc/kayobe/kolla/config/neutron/l3_agent.ini

```diff
@@ -0,0 +1,3 @@
+interface_driver = multi
+multi_interface_driver_mappings = default:openvswitch,{{ alaska_lln_physical_network }}:ipoib
```

Comment:
This option appears to need python2-networking-mlnx installed[1], but we are only installing that in neutron-server container and not neutron-base.

[1] https://opendev.org/x/networking-mlnx/src/branch/master/networking_mlnx/linux/interface_drivers/config.py#L25

Reply:
Same comment as above, not sure if this is required at all given that things are working without it.